lib/radfiledata.h

```diff
@@ -56,6 +56,7 @@ class STADIC_API RadFileData
     std::shared_ptr<RadPrimitive> addPrimitive(RadPrimitive *primitive);    //!< Add a rad primitive to the list of primitives
     std::shared_ptr<RadPrimitive> addPrimitive(std::shared_ptr<RadPrimitive> primitive);  //!< Add a rad primitive to the list of primitives
     bool setPrimitives(const shared_vector<RadPrimitive> &primitives);  //!< Replace the contents of the primitives list
+    std::shared_ptr<RadPrimitive> RadFileData::glazingsearch(std::string name);
```

Comment:
For better aliasing support I added a find function that does the same thing as this function in a better way, so lets not add this function too.